pkgs/os-specific/linux/nsncd/default.nix

```
@@ -2,21 +2,29 @@
 , stdenv
 , fetchFromGitHub
 , rustPlatform
-, nix-gitignore
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
   pname = "nsncd";
-  version = "unstable-2024-03-18";
+  version = "unstable-2024-08-06";
```

```suggestion
  version = "1.4.1-unstable-2024-04-10";
```
that is supposed to be the date of the commit

---

The commit introduced through the patch has been authored today.

See https://github.com/twosigma/nsncd/pull/119

---

We don't usually consider non-upstream patches to affect the version.

---

We don't usually consider non-upstream patches to affect the version.